_ml-commons-plugin/api.md

```diff
@@ -232,7 +232,7 @@ The API returns the following:
 
 ## Predict
 
-ML commons can predict new data with your trained model either from indexed data or a data frame. The model_id is required to use the Predict API.
+ML Commons can predict new data with your trained model either from indexed data or a data frame. The model_id is required to use the Predict API.
```

Comment:
The second sentence could be read as though The model_id itself is required to use the Predict API. How about "To use the Predict API, the model_id is required"?